test/wat_i.t

```diff
@@ -0,0 +1,335 @@
+  $ run_it() { PACMAN_LOG="$TESTDIR"/pacman.log "$TESTDIR"/../wat "$@"; }
```

Comment:
Thanks very much for adding a comprehensive test (two in fact!) but I think it's a bit redundant and could therefore turn into a maintenance burden. I think it would be OK/better to just add a single additional case to the existing test that runs with `-i` and confirms the installed stuff appears. No need to re-create all of the existing cases with `-i` and `--installed`.

If you don't want to go through the textual-hassle just say so and I'd be happy to do it myself after merging this.